lib/storage/storage.go

```diff
@@ -33,8 +33,8 @@ import (
 )
 
 const (
-	msecsPerMonth     = 31 * 24 * 3600 * 1000
-	maxRetentionMsecs = 100 * 12 * msecsPerMonth
+	retention31Days = 31 * 24 * time.Hour
```

Comment:
`retention31Days` has different meaning to `retention1Month`. 
In our docs, we operate with `month` unit. Would be nice to have constant name consistent with that